libraries/botbuilder-middleware-aws-comprehend/src/engine.ts

```diff
@@ -1,28 +1,33 @@
 import { Engine } from '@botbuildercommunity/middleware-engine-core';
-import * as aws from 'aws-sdk';
+import { AWSComprehendOptions } from './schema';
+import { AWSError, Comprehend } from 'aws-sdk';
 
 /**
  * @module botbuildercommunity/middleware-aws-comprehend
  */
 
 export class AWSComprehendEngine extends Engine {
-    public client: aws.Comprehend;
-    public constructor() {
+    public client: Comprehend;
+    public options: AWSComprehendOptions;
+    public lang: string;
+    public constructor(options?: AWSComprehendOptions) {
         super();
-        this.client = new aws.Comprehend({ apiVersion: '2017-11-27', region: 'us-east-1' });
+        this.options = options || { apiVersion: '2017-11-27', region: 'us-east-1' };
```

Comment:
Is it an idea to merge the passed options with the defaultOptions, by using the spread operator? [I have an example](https://github.com/BotBuilderCommunity/botbuilder-community-js/blob/d4c78369956129368fc290241f6983ae40a955ad/libraries/botbuilder-adapter-alexa/src/alexaAdapter.ts#L44) in the Alexa Adapter I am working on.

Reply:
Changed to spread.